cpp/harvest/nacjd.cc

```diff
@@ -64,13 +64,29 @@ namespace {
         "\t- marc_path: The output MARC file.\n"
         "\n"
         "get_statistics json_path\n"
+        "\t- json_path: The input JSON file.\n"
+        "\n"
+        "get_categories_statistics json_path\n"
         "\t- json_path: The input JSON file.\n");
 }
 
 
 std::vector<std::string> ids_website;
 const unsigned int TIMEOUT_IN_SECONDS(15);
 
+const std::string statistics_categories_array[] = { "National Crime Victimization Survey",
```

Comment:
Please add a comment here that this list contains the series from the "National Statistics" section on the following page:
https://www.icpsr.umich.edu/web/pages/NACJD/discover-data.html

Reply:
Please change the definition:
`const std::string statistics_categories_array[] = ...`
=> 
`const std::vector<std::string> statistics_categories = ...`